conference/DS/dac.yml

```diff
@@ -42,8 +42,8 @@
         id: dac25
         link: https://www.dac.com/Conference/2025-Call-for-Contributions
         timeline:
-          - abstract_deadline: '2024-12-12 17:00:00'
-            deadline: '2024-12-19 17:00:00'
+          - abstract_deadline: '2024-11-13 9:00:00'
+            deadline: '2024-11-20 9:00:00'
```

Comment:
plz get a format fix referring to https://github.com/ccfddl/ccf-deadlines/actions/runs/11051608965/job/30737693286?pr=647 ~